lib/l10n/intl_it.arb

```
@@ -831,5 +831,91 @@
+  "notification_cant_do_title": "Azione non consentita",
+  "notification_cant_do_message": "L'azione richiesta non può essere eseguita",
+  "notification_new_message_title": "Nuovo messaggio",
+  "notification_new_message_message": "Hai ricevuto un nuovo messaggio",
+  "notification_order_update_title": "Aggiornamento ordine",
+  "notification_order_update_message": "Il tuo ordine è stato aggiornato",
+
+  "@_comment_notification_details": "Etichette dei dettagli delle notifiche",
+  "buyer": "Compratore",
+  "seller": "Venditore",
+  "amount": "Importo",
+  "paymentMethod": "Metodo di Pagamento",
```

RuboCop (1.76.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)